Database.sql

```diff
@@ -0,0 +1,15 @@
+CREATE DATABASE IF NOT EXISTS FlashFeet_JohnMc;
+USE FlashFeet_JohnMc;
+
+DELIMITER $$
+DROP PROCEDURE IF EXISTS CreateDatabase $$
```

Comment:
Possibly a better name would be "CreateJobRoleTable" instead of "CreateDatabase". As on line 1 we are creating the database.